osu.Game/Overlays/Profile/Header/Components/DailyChallengeStreakTooltip.cs

```diff
@@ -0,0 +1,251 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Extensions.LocalisationExtensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Colour;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Cursor;
+using osu.Framework.Graphics.Effects;
+using osu.Framework.Localisation;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Online.API.Requests.Responses;
+using osu.Game.Scoring;
+using osuTK;
+using Box = osu.Framework.Graphics.Shapes.Box;
+using Color4 = osuTK.Graphics.Color4;
+
+namespace osu.Game.Overlays.Profile.Header.Components
+{
+    public partial class DailyChallengeStreakTooltip : VisibilityContainer, ITooltip<DailyChallengeStreakTooltipData>
+    {
+        private StreakPiece currentDaily = null!;
+        private StreakPiece currentWeekly = null!;
+        private StatisticsPiece bestDaily = null!;
+        private StatisticsPiece bestWeekly = null!;
+        private StatisticsPiece topTen = null!;
+        private StatisticsPiece topFifty = null!;
+
+        private Box topBackground = null!;
+        private Box background = null!;
+
+        [Resolved]
+        private OsuColour colours { get; set; } = null!;
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            AutoSizeAxes = Axes.Both;
+            CornerRadius = 20f;
+            Masking = true;
+
+            EdgeEffect = new EdgeEffectParameters
+            {
+                Type = EdgeEffectType.Shadow,
+                Colour = Color4.Black.Opacity(0.5f),
```

Comment:
This needs to be tempered down for sure. I'm not sure why this doesn't show on your screenshot as much but the shadow is too heavy IMO.

![1722245550](https://github.com/user-attachments/assets/6674100b-16db-48f8-a26f-65096b8058ca)

Reply:
Looks similar to what I'm seeing in my screenshot, but sure.